graph-builder/src/plugins/release_scrape_dockerv2/registry.rs

```diff
@@ -330,44 +296,62 @@ pub async fn fetch_releases(
 ///
 /// Update Images with release metadata should be immutable, but
 /// tags on registry can be mutated at any time. Thus, the cache
-/// is keyed on the hash of tag layers.
-async fn cache_release(
+/// is keyed on the manifest reference.
+#[allow(clippy::too_many_arguments)]
```

Comment:
Create a struct 3 new params?

Reply:
I don't think this is a helpful lint. Creating a struct for free-standing functions parameter seems like an anti-pattern to me, but I can be convinced otherwise.

Reply:
Agree, lets keep it and clean it up later (along with `#[allow(clippy::useless_let_if_seq)]`)

Reply:
@vrutkovs resolved?